src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -3009,12 +3009,14 @@ else if (column >= getColumnCount()) {
             for(int i = 0; i < column; i++) {
                 r.x += cm.getColumn(i).getWidth();
             }
-            //x position is updated to handled the offset to which the table
-            //has to be moved towards right side in right-to-left Orientation.
+            //Table columns are laid out from right to left when component
+            //orientation is set to ComponentOrientation.RIGHT_TO_LEFT,
+            //adjust the x coordinate for this case.
```

Comment:
```suggestion
            // Table columns are laid out from right to left when component
            // orientation is set to ComponentOrientation.RIGHT_TO_LEFT,
            // adjust the x coordinate for this case.
```
There's usually a space, it improves readability, just search for `//` in `JTable.java`.